kernel/src/kernel.rs

```diff
@@ -584,7 +584,6 @@ impl Kernel {
                             // to break to handle the interrupt, continue
                             // executing this process, or switch to another
                             // process.
-                            continue;
```

Comment:
This deletion makes me nervous.

It is technically correct, because the 150 lines of code between here at the loop end are all different match arms/etc, but it feels fragile to rely on that never changing.

It would be reasonable to imagine adding something to the "end of the kernel loop", and not see this one tiny block in the middle that expected to early-exit the loop.

I understand this was probably removed by some automated process, but I'm reticent to actually remove it. The _MoreCorrect(TM)_ thing would almost certainly be to refactor this loop into some functions or what have you, but that's a more invasive change. For the moment, I'd like to just find a way to keep this explicit control flow directive around.

Reply:
Agreed.